docs/build-on-linea/quickstart/verify-smart-contract/truffle.md

```diff
@@ -52,9 +52,9 @@ module.exports = {
         );
       },
       verify: {
-        apiUrl: "https://goerli.lineascan.build/apis#contracts",
+        apiUrl: "https://api-testnet.lineascan.build/api",        
         apiKey: ETHERSCAN_API_KEY,
-        explorerUrl: "https://goerli.lineascan.build/",
+        explorerUrl: "https://goerli.lineascan.build/address",      
```

Comment:
Do we need the `/adddress` part?
@whyvrafvr

Reply:
yes 🙏  better set it...